<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_employer</class>
 <widget class="QDialog" name="gestion_employer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
font-family:century gothic;
background:#ffb56a;
font-size:17px;

}
QGroupBox
{
background:#ffb56a;
background-image:url(:/selem2.jpg);
}
#ajouter_employer
{
background:#299a7a;
border-radius:15px;
}
#ajouter_employer:hover
{
color:#333;
border-radius:60px;
background:red;
}
#pushButton_supprimer
{
background:#299a7a;
border-radius:15px;
}
#pushButton_supprimer:hover
{
color:#333;
border-radius:60px;
background:red;
}
#nom
{
background:#299a7a;
border-radius:15px;
}
#nom:hover
{
color:#333;
border-radius:60px;
background:red;
}
#prenom
{
background:#299a7a;
border-radius:15px;
}
#prenom:hover
{
color:#333;
border-radius:60px;
background:red;
}
#id
{
background:#299a7a;
border-radius:15px;
}
#id:hover
{
color:#333;
border-radius:60px;
background:red;
}
#pushButton_modifier
{
background:#299a7a;
border-radius:15px;
}
#pushButton_modifier:hover
{
color:#333;
border-radius:60px;
background:red;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>AJOUTER</string>
    </attribute>
    <widget class="QLabel" name="label_id_employer">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID_EMPLOYER :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditidemp">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nom_employer">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NOM_EMPLOYER :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prenom_employer">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PRENOM_EMPLOYER :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mail_employer">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MAIL_EPLOYER :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditnomemp">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditprenomemp">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>140</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditmailemp">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>190</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter_employer">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>330</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>AJOUTER</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditidprofil11">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>250</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID_PROFIL :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>SUPPRIMER</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID_EMPLOYER:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_supp">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>140</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>300</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SUPPRIMER</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>AFFICHER</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>521</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>924</x>
       <y>350</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MAILING</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nom">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prenom">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>120</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="id">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditnom123">
     <property name="geometry">
      <rect>
       <x>752</x>
       <y>70</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditprenom123">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>130</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditid123">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>190</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CHERCHER PAR NOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>130</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CHERCHER PAR PRENOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>190</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CHERCHER PAR ID</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxnom">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>330</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TRIER PAR NOM</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxid">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TRIER PAR ID</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxprenom">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>330</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TRIER PAR PRENOM</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>MODIFIER</string>
    </attribute>
    <widget class="QLabel" name="label_nom_film_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NOM EMPLOYER :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prenom_employer_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PRENOM EMPLOYER :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mail_employer_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MAIL EMPLOYER :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_id_employer_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID_EMPLOYER :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditnomempM">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>90</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditprenomempM">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>150</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditmailempM">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditidempM">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>270</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

background:white

}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>140</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MODIFIER</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
